core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -191,6 +191,7 @@ public boolean isBangEqualAllowed() {
 
   @Override public boolean isPercentRemainderAllowed() {
     switch (this) {
+    case DEFAULT:
     case BABEL:
```

Comment:
You have changed the default behavior of `%` in SQL, can you give the reason ? Is it follows the SQL standard ?

Reply:
changed the default behavior of % in SQL in order to valide end-to-end IT case in JdbcTest.it may be allowed.